### PR TITLE
Cleaned up excessive whitespace

### PR DESCRIPTION
**Pull request by BaerMitUmlaut**
**When merged this pull request will:**
- Remove unnecessary whitespace in blank lines, at the end of lines and sometimes within code too (where it didn't really benefit the clarity of the code).
- Part of this is changing `_this select ...` to `params` in some AtragMX and Rangecard functions.

### CODE REVIEW

File: addons/atragmx/functions/fnc_calculate_solution.sqf [hunk 1]
```diff
@@ -42,31 +42,8 @@
...
-_targetRange        = _this select 13;
-_bc                 = _this select 14;
-_dragModel          = _this select 15;
-_atmosphereModel    = _this select 16;
-_storeRangeCardData = _this select 17;
-_stabilityFactor    = _this select 18;
-_twistDirection     = _this select 19;
-_latitude           = _this select 20;
-_directionOfFire    = _this select 21;
+params ["_scopeBaseAngle", "_bulletMass", "_boreHeight", "_airFriction", "_muzzleVelocity", "_temperature", "_barometricPressure", "_relativeHumidity", "_simSteps", "_windSpeed",                 "_windDirection", "_inclinationAngle", "_targetSpeed", "_targetRange", "_bc", "_dragModel", "_atmosphereModel", "_storeRangeCardData", "_stabilityFactor", "_twistDirection", "_latitude", "_directionOfFire"];
```
Comment (commy2) [inline addons/atragmx/functions/fnc_calculate_solution.sqf]: I suggest breaking up this very long line. There also is some strange white space in the middle.

File: addons/atragmx/functions/fnc_update_range_card.sqf [hunk 2]
```diff
@@ -43,7 +43,7 @@ _speedOfSound = GVAR(temperature) call EFUNC(weather,calculateSpeedOfSound);
     _TOF =  _x select 4;
     _velocity = _x select 5;
     _kineticEnergy = _x select 6;
```
Comment (commy2) [inline addons/atragmx/functions/fnc_update_range_card.sqf]: Suggest `_x params [` here

**Comment by commy2** [comment]
very nice, neat and tidy

File: addons/atragmx/functions/fnc_update_range_card.sqf (hunk 2 above)
Comment (BaerMitUmlaut) [inline addons/atragmx/functions/fnc_update_range_card.sqf]: I'll make another PR to find possible usage of params later, this one is mostly about whitespace.

File: addons/atragmx/functions/fnc_calculate_solution.sqf (hunk 1 above)
Comment (BaerMitUmlaut) [inline addons/atragmx/functions/fnc_calculate_solution.sqf]: Done!

**Comment by jonpas** [comment]
All :+1: from me.

**Pull request merged**